roles/ara_api/tasks/install/pypi.yaml

```diff
@@ -5,6 +5,7 @@
     state: "{{ (ara_api_version == 'latest') | ternary('latest', 'present') }}"
     version: "{{ (ara_api_version != 'latest') | ternary(ara_api_version, omit) }}"
     virtualenv: "{{ ara_api_venv | bool | ternary(ara_api_venv_path, omit) }}"
+    executable: "{{ ara_api_venv | bool | ternary(omit, '/bin/pip3') }}"
```

Comment:
I have doubts about this one because I'm not positive if the pip3 binary would always be in that location -- I can picture cases where it might be under /usr/bin/pip3 or /usr/local/bin/pip3 amongst other places.

The docs and examples in https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pip_module.html seem to indicate that we should be able to provide just ``pip3`` instead of ``/bin/pip3``. Have you tested if that works ?

Reply:
You make a good point.. Up until now I've assumed that pip is provided by the OS if not used in the venv.

The problem is that Ansible doesn't really have a PATH, so just putting ```pip3``` in there will probably break.

I'm not sure what the best way forward is, IMHO running without should be a 'last resort' (or because you're running RHEL7 that doesn't do venv's well). So maybe requiring pip3 to be provided by the OS is the best way forward.

What do you think?